server/jetstream_cluster_test.go

```diff
@@ -2145,8 +2142,9 @@ func TestJetStreamClusterMirrorAndSourceInterestPolicyStream(t *testing.T) {
 
 	checkFor(t, 5*time.Second, 250*time.Millisecond, func() error {
 		// This one will be 0 since no other interest exists.
-		if si, _ := js.StreamInfo("WQ22"); si.State.Msgs != 1 {
-			return fmt.Errorf("Expected 1 msg for %q, got %d", "WQ22", si.State.Msgs)
+		if si, _ := js.StreamInfo("IP22"); si.State.Msgs != 1 {
+			fmt.Printf("WQ22 state is %+v\n", si.State)
```

Comment:
Remove?

Reply:
Apologies, I will add tooling for this, my manual checks not doing so well lately.